shared/markdown/generate-emoji-parser.js

```diff
@@ -2,6 +2,7 @@
 import fs from 'fs'
 import path from 'path'
 import emojiData from 'emoji-datasource'
+import {escapeRegExp} from 'lodash'
```

Comment:
lodash-es

Reply:
(i'm not exactly sure why this doens't fail lint)

Reply:
lodash-es doesn't work here

Reply:
K